diff --git a/src/lib/vampimg/ImagingJpeg.pas b/src/lib/vampimg/ImagingJpeg.pas
index ef9a5e7..9af03e3 100644
--- a/src/lib/vampimg/ImagingJpeg.pas
+++ b/src/lib/vampimg/ImagingJpeg.pas
@@ -233,9 +233,9 @@ procedure JpegError(CInfo: j_common_ptr);
     // using FPC (see http://bugs.freepascal.org/view.php?id=21229).
     // Fixed in FPC 2.7.1
   {$IF Defined(FPC) and (FPC_FULLVERSION <= 20701)}
-    raise EImagingError.CreateFmt(SJPEGError + ' %d', [CInfo.err.msg_code]);
+    raise EImagingError.CreateFmt(SJPEGError + ' %d', [1]);
   {$ELSE}
-    raise EImagingError.CreateFmt(SJPEGError + ' %d: ' + string(Buffer), [CInfo.err.msg_code]);
+    raise EImagingError.CreateFmt(SJPEGError + ' %d: ' + string(Buffer), [1]);
   {$IFEND}
   end;
