diff --git a/Makefile b/Makefile
index fcbed5d226..e45dc43f72 100644
--- a/Makefile
+++ b/Makefile
@@ -369,7 +369,7 @@ endif
 override PACKAGE_NAME=fpc
 override PACKAGE_VERSION=3.3.1
 REQUIREDVERSION=3.2.2
-REQUIREDVERSION2=3.2.0
+REQUIREDVERSION2=3.3.1
 ifndef inOS2
 override FPCDIR:=$(BASEDIR)
 export FPCDIR
diff --git a/Makefile.fpc b/Makefile.fpc
index c03a7dccc0..14d0592f72 100644
--- a/Makefile.fpc
+++ b/Makefile.fpc
@@ -21,7 +21,7 @@ rule=help
 
 [prerules]
 REQUIREDVERSION=3.2.2
-REQUIREDVERSION2=3.2.0
+REQUIREDVERSION2=3.3.1
 
 
 # make versions < 3.77 (OS2 version) are buggy
